src/components/Gradebook/footer.jsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { Hyperlink, Icon } from '@edx/paragon';
 
-import EdXLogo from '../../../assets/edx-sm.png';
+import EdxFooterLogo from '../../../assets/edx-footer.png';
```

Comment:
Tiny: Arguably, EdXFooterLogo (upper case X) is the more correct camel casing. 🐫